DiscordCode/BackEnd/routes/Models/Server.model.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema
+
+const UserSchema = new Schema({
```

Comment:
ServerModel has already been created and implemented for the ServerPage, maybe this could be turned into a strictly Server Settings model with sid being a key for both the server and server settings